doc/Development_Documentation/05_Objects/03_Working_with_PHP_API.md

```diff
@@ -238,6 +247,9 @@ foreach ($list as $city) {
 
 ```php
 $list = DataObject\News::getByLocalizedfields($fieldName, $value, $locale, $limit | array($limit, $offset, $unpublished));
+
+// or
+$list = DataObject\News::getByFieldName($value, $locale, $limit | array($limit, $offset, $unpublished));
```

Comment:
@BlackbitNeueMedien could you please fix the wrong example here for last parameter as array? (AFAIK it is not possible) and also for `getByLocalizedfields`.

Or if possible, changes to support it? Thanks!

Reply:
Fixed.
Only the example in the docs was wrong. `getByLocalizedfields()` also works with list config as 4th parameter - but it has to be an associative array.

Reply:
ohh got it,..😄 thanks for your changes!